lib/hanami/events/subscriber.rb

```diff
@@ -1,20 +1,15 @@
-require 'hanami/events/handler_runner'
-
 module Hanami
   module Events
-    class Subscriber
-      def initialize(event_name, event_handler, logger = nil)
-        @event_name = event_name
-        @pattern_matcher = Matcher.new(event_name)
-        @handler_runner = HandlerRunner.new(event_handler, logger)
-      end
-
-      def call(event_name, payload)
-        @handler_runner.(payload) if @pattern_matcher.match?(event_name)
+    module Subscriber
+      def self.included(klass)
+        klass.extend(ClassMethods)
       end
 
-      def meta
-        { name: @event_name }
+      module ClassMethods
+        def subscribe_to(event_bus, event_name)
+          handler = self
```

Comment:
it's not a handler. I think we can use `klass` here or change to the instance like: `object = new`. Or something like this